### ISSUE TITLE
Update a11y for back/cancel/done/save buttons

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#17185]
## Dev Task 
1. Update a11y for the back/cancel/done/save buttons to be more generic
     - NOTE: Custom a11y will be noted in individual accessibility ACs

## Accessibility Acceptance Criteria
1. Everything on the page should be read by the screen reader.
     - If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)
2. Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap, or horizontal scroll.
3. Each touch target must be at minimum 44px by 44px
4. The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.  
5. All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
6. When a page opens, the first section or element should automatically be placed in focus
7. When a page opens, the title page should be announced

## Accessibility Implementation Guidelines 
**Back Button**
- accessibilityLabel= “back”
- accessibilityHint= “Navigates to the previous page” 
- accessibilityRole= “button”

**Cancel Button**
- accessibilityLabel= "cancel"
- accessibilityHint= "Cancels changes and navigates to the previous page"
- accessibilityRole= “button

**Save Button**
- accessibilityLabel= "save"
- accessibilityHint= "Saves changes and navigates to the previous page"
- accessibilityRole= “button”

**Done Button**
- accessibilityLabel= "done"
- accessibilityHint= “Exits out of the web view and navigates to the previous page” 
- accessibilityRole= “button

**Comment by digitalken** [comment]
Added to TestRail.

**Comment by digitalken** [comment]
Testing...
